subsquid/test/bondedFinanceTest.ts

```diff
@@ -238,3 +304,53 @@ describe("Bonded finance events", () => {
     verify(storeMock.save(anyOfClass(BondedFinanceBondOffer))).times(3);
   });
 });
+
+describe("Vesting schedule added", () => {
```

Comment:
I would move these tests to a dedicated file for `vesting`

Reply:
✅